common/pkcs11.h

```diff
@@ -856,6 +958,61 @@ struct ck_rsa_pkcs_oaep_params {
   unsigned long source_data_len;
 };
 
+struct ck_aes_ctr_params {
+  unsigned long ulCounterBits;
```

Comment:
nit: we use the GNU coding convention for structure members; see `ck_rsa_pkcs_pss_params.hash_alg` and [the renaming macro](https://github.com/p11-glue/p11-kit/pull/181/commits/cf669c242a1a9a646128d65440edf883a00a36fd#diff-79ec9ebeae42b2c808fb40814768ecc3R191) for example.

Reply:
Sorry for not responding earlier; it's not a blocker (just looked different from other part of the header).  Given that the only consumer of this convention is GnuTLS and it doesn't (yet) use the newly added structs, I guess it would be fine to keep the original names.